### PR TITLE
Final Milestone

### PR DESCRIPTION
**Pull request by ccobasi**
- Implement add task function
- Implement delete task function
- Implement clear all function

### CODE REVIEW

File: src/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <span>
+      <input class='check' type='checkbox' id='task-description' name='task-description' value='${task}'>${task}
+      <i class="fas fa-ellipsis-v ellipsis" style="color: gray; float: right;"></i>
+      </span>
+      `;
+  todoList.appendChild(newTask);
+  saveChanges();
+  saveLocalstorage(list);
+  document.location.reload(true);
+}
```
Comment (Davidosky007) [inline src/addRemove.js]: * Currently only one task can be added in the `list` array,  so as a result other features[[These features](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new,in%20local%20storage.)] of this milestone cant be tested kindly fix this 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @ccobasi 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Kindly check below and make the necessary changes to the List `array` in the local storage, it should be able to hold more tasks. 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @ccobasi 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ccobasi** [comment]
Have not merged PR

**Pull request merged**